Bio/FSSP/__init__.py

```diff
@@ -95,14 +95,15 @@ def __init__(self, inStr):
             else:
                 self.ss = '0'
 
-    def __repr__(self):
+    def __str__(self):
```

Comment:
I have read that if `__repr__` is defined and `__str__` not, `__str__` would use `__repr__` automatically.

I guess you have a good reason to do it the more 'complicated' way?

Reply:
You're right, good point: https://docs.python.org/2/reference/datamodel.html

Let's define just ``__repr__`` here then (and remove the historic duplication)

Reply:
Nope, just didn't like the historic redundant double definition.